helm/aqua-app-server/values.schema.json

```diff
@@ -29,73 +29,73 @@
                     "type": "boolean"
                 },
                 "dbAuditPasswordKey": {
-                    "type": ["null", "string"]
+                    "type": "null"
```

Comment:
```suggestion
                    "type": ["null", "string"]
```

Reply:
this is just an example - there shouldn't be a bare `null` type anywhere in that file